tests/e2e/specs/shopper/cart-checkout/translations.test.js

```diff
@@ -34,12 +34,13 @@ describe( 'Shopper → Cart & Checkout → Translations', () => {
 		const removeLink = await page.waitForSelector(
 			'.wc-block-cart-item__remove-link'
 		);
-		await expect( removeLink ).toMatch( 'Artikel verwijderen' );
+		await expect( removeLink ).toMatch( 'Verwijder item' );
```

Comment:
This translation changed in recent releases.

Reply:
@senadir do you see the translation of `Remove item` on your end? In my test, other strings work but the translation of the remove link wasn't loaded. Tested with this PR, `trunk`, and the .org release.

<img width="1616" alt="Screen Shot 2022-05-19 at 14 19 25" src="https://user-images.githubusercontent.com/5423135/169236332-091f4fe9-37b0-486e-9db2-98a9bb032538.png">

Reply:
I just tested this PR and I see the translation for the remove link 
![image](https://user-images.githubusercontent.com/4463174/169785602-b8b30d34-2be8-4f59-ae32-2fbf919b5823.png)

Reply:
Tung, in your local branch, did you update your translations by using make-pot and make-json?